### ISSUE TITLE
kubedctl enable services has conflict with knative services

### ISSUE THREAD

**Issue by qiujian16** [kubernetes-sigs/kubefed#1078]
**What happened**:

run `$ ./bin/kubefedctl enable services --kubefed-namespace default`
returns `
F0802 15:16:29.293248   61760 enable.go:110] Error: Multiple resources are matched by "services": services, services.serving.knative.dev. A group-qualified plural name must be provided.`

The reason is that i installed knative services crd before. However, we cannot enable kube services using command line, since it has no group.

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

install knative services and run kubefedctl enable services

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`)
- KubeFed version
- Scope of installation (namespaced or cluster)
- Others

/kind bug

**Comment by marun** [comment]
@qiujian16 Good catch.  The 'core' API types lack of a group doesn't allow providing a group-qualified name in cases like this where the kind/plural name is ambiguous.  Would it make sense for `kubefedctl enable` to accept `core` as a qualifier (e.g. `services.core`) for these types and then strip the suffix before use?

**Comment by qiujian16** [comment]
for better user experience, should we have `kubefedctl enable service` points to the v1/services instead of running `kubefedctl enable service.core`? It is similar for federated deployment, today we need to run `kubefedctl enable deployment.apps` or `kubefedctl enable deployment.extensions`, but it will be better if we can run `kubefedctl enable deployment` which then points to a certain api group. WDYT?

**Comment by marun** [comment]
@qiujian16 The current approach is requiring that the user be explicit, and I don't think this case suggests deviating from that approach. We can't assume that we know better than the user as to what they should be enabling.

**Comment by qiujian16** [comment]
yes..that makes sense. So if there are not multiple match, should still use `kubefedctl enable service`, otherwise, should use `kubefedctl enable service.core`

**Comment by marun** [comment]
And not just for services - kubefedctl should accept `*.core `as a type name and strip the `.core` suffix for interaction with the API.

**Comment by ysjjovo** [comment]
So what's the solution?
I still got error message while execute command ```kubefedctl enable service.core```.
```
F0904 16:14:12.809596   49450 enable.go:111] Error: Unable to find api resource named "service.core".
```

**Comment by makkes** [comment]
I issued a PR with a potential fix: #1294
